src/installing-deis/quickstart.md

```diff
@@ -14,6 +14,18 @@ Choose one of the following providers and deploy a new kubernetes cluster:
 - [Google Container Engine](https://cloud.google.com/container-engine/docs/before-you-begin)
 - [Vagrant](http://kubernetes.io/v1.1/docs/getting-started-guides/vagrant.html)
 
+## Prerequisites
+Please make sure you enable the Daemon Sets api if you are installing a pre-1.2 version of kubernetes. As it is not turned on by default. You can learn more about how to do that [here](https://github.com/kubernetes/kubernetes/blob/master/docs/api.md#enabling-resources-in-the-extensions-group).
```

Comment:
I feel like we should reference the official documentation at http://kubernetes.io/v1.1/docs/api.html#enabling-resources-in-the-extensions-group